### DISCUSSION TITLE
SVG Thumbnails

### DISCUSSION THREAD

**Discussion by skygatewitt** [pimcore/pimcore#15598] [Q&A / Support]
Hi guys,

I'm trying to figure out, how to best setup Pimcore's thumbnails to work both with rasterized images as well as SVG.

Using the default thumbnails with SVG assets and the format set to "Auto (Web-optimized - recommended)" results in blurry, cropped, rasterized images (unless I also activate "Preserve animations for GIF (Imagick)", in which case I got the filetype-not-supported placeholder).

I know there is a format "Print", but I don't fancy duplicating all my thumbnail configs to have a print version of each. 

I was able to make it work by generating the thumbnail in Twig and setting the format to "print" based on the asset's file extension before calling getHtml(). But to do that in every template using thumbnails seems a bit overkill-y. 

Is there a better way to tell Pimcore to just leave the SVG alone and render them as-is (but adding the width and height to the image tag), while still retaining the thumbnailing for rasterized images?

Thank you!

**Comment by dvesh3** [comment]
Hi @skygatewitt you can try changing `auto-formats` quality in the config https://pimcore.com/docs/platform/Pimcore/Assets/Working_with_Thumbnails/Image_Thumbnails#customize-auto-web-optimized-format
